Dockerfile

```diff
@@ -1,60 +1,24 @@
+ 
 FROM gcr.io/deeplearning-platform-release/tf2-gpu.2-2
 WORKDIR /root
 
 # Installs sndfile library for reading and writing audio files 
 RUN apt-get update && apt-get install -y libsndfile-dev
```

Comment:
add --no-installs-recommends to the apt-get command so it doesn't install extra unwanted packages.